frb_spectral_search/acf_func.py

```diff
@@ -122,7 +122,7 @@ def acf_bands(spectra, n, plot_range = None, band_size = 40, normalize = True, s
     fig, ax = plt.subplots(1, 1)
     start = 0 #counts channels of bands
     n2 = len(fig.axes)
-    freqs = np.linspace(800, 400, 1024*16)
```

Comment:
Because CHIME/FRB drops the Nyquist frequency of 400.0 MHz (the last 1k channel is centered at 400.390625 MHz), it's necessary to use `endpoint=False` in this idiom so you don't also get (the 1025th) channel centered 400 MHz.